crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -60,14 +77,21 @@ impl Plugin for SpritePickingPlugin {
 )]
 fn sprite_picking(
     pointers: Query<(&PointerId, &PointerLocation)>,
-    cameras: Query<(Entity, &Camera, &GlobalTransform, &Projection)>,
+    cameras: Query<(
+        Entity,
+        &Camera,
+        Option<&SpritePickable>,
+        &GlobalTransform,
+        &Projection,
+    )>,
     primary_window: Query<Entity, With<PrimaryWindow>>,
     images: Res<Assets<Image>>,
     texture_atlas_layout: Res<Assets<TextureAtlasLayout>>,
     settings: Res<SpritePickingSettings>,
     sprite_query: Query<(
         Entity,
         &Sprite,
+        Option<&SpritePickable>,
```

Comment:
Don't we need to use a `With` filter to actually get the desired performance benefits?